Document connection(...) requirement in makeExtendSchemaPlugin (#2352)
graphile/crystal: postgraphile/website/postgraphile/make-extend-schema-plugin.md

});
```

> ## Returning Connection

Reviewer: ```suggestion
## Returning a connection

```

> When returning a connection in `makeExtendSchemaPlugin`, you must wrap the return value with `connection(...)`.  

Reviewer: ```suggestion
When returning a connection in `makeExtendSchemaPlugin`, you must be sure that your plan resolver yields a `connection(...)` step; failure to do this may result in errors such as `$connection.getSubplan is not a function`.
```

For example, if you want to expose a related `ReviewConnection` from `Product`:

```js
import { makeExtendSchemaPlugin, gql } from "postgraphile/utils";
import { connection } from 'postgraphile/grafast';

export const MyProductReviewsPlugin = makeExtendSchemaPlugin((build) => {
  const { reviews } = build.input.pgRegistry.pgResources;

  return {
    typeDefs: gql`
      extend type Product {
        reviews: ReviewConnection
      }
    `,
    plans: {
      Product: {

>         reviews() {
>           // highlight-next-line
>           return connection(reviews.find());

Reviewer: ```suggestion
        reviews($product) {
          const $productId = $product.get('id');
          const $reviews = reviews.find();
          $reviews.where(sql`${$reviews}.product_id = ${$productId}`);

          // highlight-next-line
          return connection($reviews);
```
